integration/docker/info_test.go

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2018 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package docker
+
+import (
+	. "github.com/kata-containers/tests"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("info", func() {
+	var (
+		stdout   string
+		exitCode int
+	)
+
+	Context("docker info", func() {
+		It("should has a runtime information", func() {
+			stdout, _, exitCode = dockerInfo()
+			Expect(exitCode).To(Equal(0))
+			Expect(stdout).To(ContainSubstring("Runtimes: " + Runtime))
```

Comment:
I think we should make this check more reliable using a regex, in case there are more runtimes registered in docker. 
We might have in the same configuration: `cc-runtime`, `runv` and `runc`.